modules/virt-creating-configmap.adoc

```diff
@@ -150,13 +250,13 @@ parameters:
 EOF
 ----
 
-. Verify that the Windows template was created by running the following command:
+.. Verify that the Windows template was created:
```

Comment:
I think we're still using "by running the following command"

Reply:
I took that out because it's too repetitious. Not everyone introduces a command that way. Also, IBM, 186, says not to use 'run' for commands in procedures.